[V5.2] PHP linters did not run locallly with PHP 8

**Describe the bug**
When you run MegaLinter locally with docker images, on source code installed for PHP 8, all PHP linters did not run correctly

**Expected behavior**
Be able to specify which PHP built-in version (7.4 or 8.0) to run for each PHP linter (PHPStan, PHPCS, Psalm, PHPLint)

**Screenshots**
```
Results of phpstan linter (version 1.2.0)
See documentation on https://megalinter.github.io/descriptors/php_phpstan/
-----------------------------------------------

❌ [ERROR] for workspace /tmp/lint
Linter raw log:
PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.0". You are running 7.4.26. in /tmp/lint/vendor/composer/platform_check.php on line 24
Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.0". You are running 7.4.26. in /tmp/lint/vendor/composer/platform_check.php on line 24

```

**Additional context**
```
----------------------------------------------------------------------------------------------------
-------------------------------------------- MegaLinter --------------------------------------------
----------------------------------------------------------------------------------------------------
 - Image Creation Date: 2021-12-03T06:10:37Z
 - Image Revision: e770f95f
 - Image Version: beta
```

Run with following command : `sudo rm -rf report/ && time docker run -v $(pwd):/tmp/lint -w $(pwd) megalinter/megalinter-php:beta`

--- a/Dockerfile
+++ b/Dockerfile
@@ -105,6 +105,7 @@ RUN apk add --update --no-cache \
                 php8-dom \
                 php8-simplexml \
                 composer \
+                dpkg \
                 ruby \
                 ruby-dev \
                 ruby-bundler \
@@ -267,6 +268,8 @@ RUN wget --tries=5 -q -O phive.phar https://phar.io/releases/phive.phar \
     && chmod +x phive.phar \
     && mv phive.phar /usr/local/bin/phive \
     && rm phive.phar.asc
+RUN update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
+    && update-alternatives --install /usr/bin/php php /usr/bin/php8 10
 
 
 # POWERSHELL installation

--- a/Dockerfile
+++ b/Dockerfile
@@ -267,8 +267,8 @@ RUN wget --tries=5 -q -O phive.phar https://phar.io/releases/phive.phar \
     && gpg --verify phive.phar.asc phive.phar \
     && chmod +x phive.phar \
     && mv phive.phar /usr/local/bin/phive \
-    && rm phive.phar.asc
-RUN update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
+    && rm phive.phar.asc \
+    && update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
     && update-alternatives --install /usr/bin/php php /usr/bin/php8 10
 
 

--- a/flavors/php/Dockerfile
+++ b/flavors/php/Dockerfile
@@ -185,8 +185,8 @@ RUN wget --tries=5 -q -O phive.phar https://phar.io/releases/phive.phar \
     && gpg --verify phive.phar.asc phive.phar \
     && chmod +x phive.phar \
     && mv phive.phar /usr/local/bin/phive \
-    && rm phive.phar.asc
-RUN update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
+    && rm phive.phar.asc \
+    && update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
     && update-alternatives --install /usr/bin/php php /usr/bin/php8 10
 
 

--- a/flavors/php/Dockerfile
+++ b/flavors/php/Dockerfile
@@ -83,6 +83,7 @@ RUN apk add --update --no-cache \
                 php8-dom \
                 php8-simplexml \
                 composer \
+                dpkg \
                 ansible \
                 ansible-lint \
                 libc-dev \
@@ -185,6 +186,8 @@ RUN wget --tries=5 -q -O phive.phar https://phar.io/releases/phive.phar \
     && chmod +x phive.phar \
     && mv phive.phar /usr/local/bin/phive \
     && rm phive.phar.asc
+RUN update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
+    && update-alternatives --install /usr/bin/php php /usr/bin/php8 10
 
 
 # actionlint installation

--- a/megalinter/descriptors/php.megalinter-descriptor.yml
+++ b/megalinter/descriptors/php.megalinter-descriptor.yml
@@ -26,6 +26,7 @@ install:
     - php8-dom
     - php8-simplexml
     - composer
+    - dpkg
   dockerfile:
     - |
       RUN wget --tries=5 -q -O phive.phar https://phar.io/releases/phive.phar \
@@ -39,6 +40,8 @@ install:
           && chmod +x phive.phar \
           && mv phive.phar /usr/local/bin/phive \
           && rm phive.phar.asc
+      RUN update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
+          && update-alternatives --install /usr/bin/php php /usr/bin/php8 10
 linters:
   # PHP Built in Linter
   - linter_name: php

--- a/megalinter/descriptors/php.megalinter-descriptor.yml
+++ b/megalinter/descriptors/php.megalinter-descriptor.yml
@@ -39,8 +39,8 @@ install:
           && gpg --verify phive.phar.asc phive.phar \
           && chmod +x phive.phar \
           && mv phive.phar /usr/local/bin/phive \
-          && rm phive.phar.asc
-      RUN update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
+          && rm phive.phar.asc \
+          && update-alternatives --install /usr/bin/php php /usr/bin/php7 100 \
           && update-alternatives --install /usr/bin/php php /usr/bin/php8 10
 linters:
   # PHP Built in Linter

